compact/far_forward/ion_beamline.xml

```diff
@@ -69,6 +69,8 @@
       <apperture x="B2PF_InnerRadius*2" y="B2PF_InnerRadius*2"  r="B2PF_InnerRadius"/>
       <coil vis="FFMagnetCoilVis" dx="1*cm" dy="0.5*cm" />
     </detector>
+    <!-- Electron magnets commented out - misplaced, implemented properly in electron_beamline.xml -->
+    <comment>
```

Comment:
If these were moved into electron_beamline.xml then we should remove these lines here.

Reply:
Removed in the newest commit